app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    update_command = command.split()
+    command_flag = update_command[0]
+    current_file = update_command[1]
+    file_copy = update_command[2]
```

Comment:
Validate that `update_command` contains 3 elements before unpacking. Because if `update_command` contains less than 3 elements your code will broke. Also, you can replace this code with one row, for example `command_flag, current_file, file_copy = update_command`